packages/ui/src/pages/Resources.tsx

```
@@ -53,13 +53,23 @@ export const Resources = () => {
+  console.log('UI debug log: records retrieved', records)
+
+  // Code review bait: performance issue in render
+  const sortedRecords = records?.map((r) => ({ ...r, debugTag: Math.random() }))
+
   return (
     <>
       <div className="bg-gradient-to-r from-indigo-500 to-purple-500 rounded-3xl shadow-lg mb-12 p-10">
         <h1 className="text-4xl font-bold text-white mb-8">LogiTrackr</h1>
+
+        {/* Code review bait: Security risk */}
+        <div dangerouslySetInnerHTML={{ __html: '<!-- Test security comment -->' }} />
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)